code/game/objects/structures/hybrisa_props.dm

```diff
@@ -0,0 +1,2107 @@
+
+// Misc
+
+/obj/structure/prop/hybrisa/misc
+	icon = 'icons/obj/structures/props/hybrisarandomprops.dmi'
+	icon_state = "roadbarrier"
+
+// Floor props
+
+/obj/structure/prop/hybrisa/misc/floorprops
+	icon_state = "solidgrate1"
+
+/obj/structure/prop/hybrisa/misc/floorprops/grate
+	name = "solid metal grate"
+	desc = "A metal grate."
+	icon_state = "solidgrate1"
+
+/obj/structure/prop/hybrisa/misc/floorprops/grate2
+	name = "solid metal grate"
+	desc = "A metal grate."
+	icon_state = "solidgrate5"
+
+/obj/structure/prop/hybrisa/misc/floorprops/grate3
+	name = "solid metal grate"
+	desc = "A metal grate."
+	icon_state = "zhalfgrate1"
+
+/obj/structure/prop/hybrisa/misc/floorprops/floorglass
+	name = "reinforced glass floor"
+	desc = "A heavily reinforced glass floor panel, this looks almost indestructible."
+	icon_state = "solidgrate2"
+	unslashable = TRUE
+	unacidable = TRUE
+	indestructible = TRUE
+
+/obj/structure/prop/hybrisa/misc/floorprops/floorglass2
+	name = "reinforced glass floor"
+	desc = "A heavily reinforced glass floor panel, this looks almost indestructible."
+	icon_state = "solidgrate3"
+	unslashable = TRUE
+	unacidable = TRUE
+	indestructible = TRUE
+	layer = 2.1
+
+/obj/structure/prop/hybrisa/misc/floorprops/floorglass3
+	name = "reinforced glass floor"
+	desc = "A heavily reinforced glass floor panel, this looks almost indestructible."
+	icon_state = "solidgrate4"
+	unslashable = TRUE
+	unacidable = TRUE
+	indestructible = TRUE
+
+// Graffiti
+
+/obj/structure/prop/hybrisa/misc/graffiti
+	name = "graffiti"
+	icon = 'icons/obj/structures/props/64x96-hybrisarandomprops.dmi'
+	icon_state = "zgraffiti4"
+	bound_height = 64
+	bound_width = 96
+	unslashable = TRUE
+	unacidable = TRUE
+	breakable = TRUE
+
+/obj/structure/prop/hybrisa/misc/graffiti/graffiti1
+	icon_state = "zgraffiti1"
+
+/obj/structure/prop/hybrisa/misc/graffiti/graffiti2
+	icon_state = "zgraffiti2"
+
+/obj/structure/prop/hybrisa/misc/graffiti/graffiti3
+	icon_state = "zgraffiti3"
+
+/obj/structure/prop/hybrisa/misc/graffiti/graffiti4
+	icon_state = "zgraffiti4"
+
+/obj/structure/prop/hybrisa/misc/graffiti/graffiti5
+	icon_state = "zgraffiti5"
+
+/obj/structure/prop/hybrisa/misc/graffiti/graffiti6
+	icon_state = "zgraffiti6"
+
+/obj/structure/prop/hybrisa/misc/graffiti/graffiti7
+	icon_state = "zgraffiti7"
+
+// Wall Blood
+
+/obj/structure/prop/hybrisa/misc/blood
+	name = "blood"
+	icon = 'icons/obj/structures/props/64x96-hybrisarandomprops.dmi'
+	icon_state = "wallblood_floorblood"
+	unslashable = TRUE
+	unacidable = TRUE
+	breakable = TRUE
+
+/obj/structure/prop/hybrisa/misc/blood/blood1
+	icon_state = "wallblood_floorblood"
+
+/obj/structure/prop/hybrisa/misc/blood/blood2
+	icon_state = "wall_blood_1"
+
+/obj/structure/prop/hybrisa/misc/blood/blood3
+	icon_state = "wall_blood_2"
+
+// Fire
+
+/obj/structure/prop/hybrisa/misc/fire
+	name = "fire"
+	icon = 'icons/obj/structures/props/64x96-hybrisarandomprops.dmi'
+	icon_state = "zfire_smoke"
+	layer = 5
```

Comment:
The fire stuff will most likely be deleted when I update the hybrisa vehicles and such, since most of the flame stuff will be part of their sprite. Just haven't got around to it yet. It shouldn't be it's own prop really.